### PR TITLE
install cate-jl-ext in the jupyterhub image

### PR DESCRIPTION
**Pull request by TejasMorbagal**
This PR allows to install cate-jl-ext in the jupyterhub image. This dockerfile is intended to build stage and production images

### CODE REVIEW

**Review by TonioF — CHANGES_REQUESTED** [review]
Can you exchange the occurrences of python setup.py install with pip install -e . , please?

**Comment by TejasMorbagal** [comment]
> Can you exchange the occurrences of python setup.py install with pip install -e . , please?

Done :+1:

**Review by TonioF — APPROVED** [review]

File: .env [hunk 1]
```diff
@@ -8,4 +8,7 @@ XCUBE_VERSION=0.13.0
 XCUBE_CCI_INSTALL_MODE=release
 XCUBE_CCI_VERSION=0.9.8
 
-INSTALL_MOOC=0
\ No newline at end of file
+INSTALL_MOOC=0
```
Comment (forman) [inline .env:11]: Is this needed? If not, please remove.

**Review by forman — COMMENTED** [review]
Please check use of `INSTALL_MOOC`. Else good.

File: .env (hunk 1 above)
Comment (TonioF) [reply .env:11]: MOOC notebooks are not needed, but this switch is set to not install anyway.

Comment (TejasMorbagal) [reply .env:11]: Yes, I will keep this flag in case we require this later

**Pull request merged**